openlibrary/plugins/worksearch/languages.py

```diff
@@ -40,19 +42,38 @@ def process_key(self, key):
         return key.replace("_", " ")
 
 
+def get_top_languages(limit):
+    from . import search
```

Comment:
Can this be imported once at global scope?

Reply:
Looks like this was perhaps for a previous PR? If I recall correctly, I think I did that because it was causing a dependency loop  :/ But can't recall.

Reply:
Closing as won't fix; I can't recall the details and no longer part of this PR.